library/riscv-aia/src/peripheral/aplic.rs

```diff
@@ -481,6 +482,8 @@ impl IntTarget {
     }
 
     /// Set interrupt priority.
+    /// Note: According to RISC-V AIA specification, priority value 0 is reserved.
+    /// Hardware automatically converts priority 0 to 1 when writing to this field.
```

Comment:
```suggestion
    /// Set interrupt priority.
    ///
    /// Note: According to RISC-V AIA specification, priority value 0 is reserved.
    /// Hardware automatically converts priority 0 to 1 when writing to this field.
```
Either add a new line, or convert the 'Note' part into conventional comment (`//`) instread of documentation comment (`///`).